Build failure on Sierra (10.12.2) due to libqwt-dev dependency

```
brew install gazebo8
==> Installing gazebo8 from osrf/simulation
==> Downloading http://gazebosim.org/distributions/gazebo/releases/gazebo-8.0.0.tar.bz2
Already downloaded: /Users/salbert/Library/Caches/Homebrew/gazebo8-8.0.0.tar.bz2
==> Downloading https://gist.githubusercontent.com/scpeters/9199370/raw/afe595587e38737c537124a3652db99de026c272/brew_python_fix.patch
Already downloaded: /Users/salbert/Library/Caches/Homebrew/gazebo8--patch-c4774f64c490fa03236564312bd24a8630963762e25d98d072e747f0412df18e.patch
==> Patching
==> Applying brew_python_fix.patch
patching file deps/opende/CMakeLists.txt
==> cmake .. -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/opt/brew/Cellar/gazebo8/8.0.0_2 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev
Last 15 lines from /Users/salbert/Library/Logs/Homebrew/gazebo8/01.cmake:
-- 	Graphviz not found, Model editor's schematic view will be disabled.
-- 	OpenSource Virtual Reality (OSVR) support will be disabled.
-- END BUILD WARNINGS

-- BUILD ERRORS: These must be resolved before compiling.
-- 	Missing: libqwt-dev. Required for plotting.
-- END BUILD ERRORS

CMake Error at CMakeLists.txt:303 (message):
  Errors encountered in build.  Please see the BUILD ERRORS above.


-- Configuring incomplete, errors occurred!
See also "/tmp/gazebo8-20170203-95067-1g1mcoo/gazebo-8.0.0/build/CMakeFiles/CMakeOutput.log".
See also "/tmp/gazebo8-20170203-95067-1g1mcoo/gazebo-8.0.0/build/CMakeFiles/CMakeError.log".
```

qwt-6.1.3_3 is installed according to Homebrew.

I would post the CMake logs, but I don't know how to keep Homebrew from deleting them.

--- a/gazebo8.rb
+++ b/gazebo8.rb
@@ -63,6 +63,8 @@ class Gazebo8 < Formula
     ENV.m64
 
     cmake_args = std_cmake_args
+    cmake_args << "-DQWT_WIN_INCLUDE_DIR=#{HOMEBREW_PREFIX}/lib/qwt.framework/Headers"
+    cmake_args << "-DQWT_WIN_LIBRARY_DIR=#{HOMEBREW_PREFIX}/lib/qwt.framework"
 
     mkdir "build" do
       system "cmake", "..", *cmake_args

